raiden/tests/scenarios/ci/sp2/bf6_stress_hub_node.yaml

```diff
@@ -41,6 +41,7 @@ nodes:
   node_options:
     0:
       matrix-server: https://transport01.raiden.network
+      switch-tracing: true
```

Comment:
What does this do?

Reply:
It enables the greenlet switch logger: https://github.com/raiden-network/raiden/blob/develop/raiden/utils/profiling/greenlets.py

But I already removed this locally.